README.md

```diff
@@ -1,44 +1,50 @@
+## DO NOT ADD ANY EXTERNAL LIBRARIES
 
-### `npm start`
+Everything needed to develop the project has been setup properly including routing, css libraries, liniting files, build process. DO NOT add any external library without discussion. Please simply take a pull from develop and start adding HTML, CSS and TypeScript Code.
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Collaboration Guidelines :checkered_flag:
 
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
+### Code Quality :art:
 
-### `npm test`
+* No class based Components. Use [hooks](https://reactjs.org/docs/hooks-overview.html#state-hook) only.
+* No implicit types. To ensure the code is of the highest quality, use explicit types when declaring interfaces.
+* Always develop the individual component in isolation using [React Storybook](https://storybook.js.org/). Do not add components directly to app. Check for all possible inputs and scenarios in storybook first.
+* Always use [code splitting](https://facebook.github.io/create-react-app/docs/code-splitting) using ```React.lazy``` and ```React.suspense```
+* Make sure to use [error boundaries](https://reactjs.org/docs/error-boundaries.html#introducing-error-boundaries)
+* Justify every single line of code you write. If you don't know what your code is doing or why did you include it, it will be removed.
+* Use [CSS modules](https://facebook.github.io/create-react-app/docs/adding-a-css-modules-stylesheet) only. No global CSS is allowed. Every component should be responsible for its own presentation.
 
-Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### Testing :white_check_mark:
 
-### `npm run build`
+* The testing suite has been configured and setup. You just need to write your tests.
+* A combination of [JEST](https://jestjs.io/) and [React Testing library](https://testing-library.com/docs/react-testing-library/intro) should be used for component testing.
+* [Cypress](https://www.cypress.io/) will be used for end-to-end testing.
+* Pull Request for individual components will not be merged with passing JEST tests.
+* Pull Request for completed features will not be merged without end-to-end testing with Cypress.
 
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### Project structure :card_file_box:
 
-The build is minified and the filenames include the hashes.<br>
-Your app is ready to be deployed!
+* The project directories has been setup using official react guidelines found on [this page](https://reactjs.org/docs/faq-structure.html). Please be sure to follow to directory structure to ensure that everything can be located by at a single glance.
```

Comment:
That link only contains some information without strict guidelines. You need to specify which of the 2 approaches you chose and why.